### DISCUSSION TITLE
☂️ Feature request: Add option to keep the import statement always on one line

### DISCUSSION THREAD

**Discussion by Waterstraal** [rome/tools#3725] [Suggestions]
### Description

**Describe the feature**
I would like to have an option that forces imports to be on 1 line. 

So this:
```ts
import huffman from "n-ary-huffman";
import { repr } from "tiny-decoders";

import iconsChecksum from "../icons/checksum";
import type {
  ElementReport,
  ElementTypes,
  ElementWithHint,
  ExtendedElementReport,
  HintMeasurements,
  HintUpdate,
} from "../shared/hints";
import {
  type HintsMode,
  type KeyboardAction,
  type NormalizedKeypress,
} from "../shared/keyboard";
import {
  Resets,
  addListener,
  bind,
  isMixedCase,
  log,
  makeRandomToken,
  partition,
  splitEnteredText,
  unreachable,
} from "../shared/main";
import type {
  FromBackground,
  FromOptions,
  FromPopup,
  FromRenderer,
  FromWorker,
  ToBackground,
  ToOptions,
  ToPopup,
  ToRenderer,
  ToWorker,
} from "../shared/messages";
import {
  type OptionsData,
  type PartialOptions,
  flattenOptions,
  getDefaults,
  makeOptionsDecoder,
  unflattenOptions,
} from "../shared/options";
import { type Durations, type Perf, TimeTracker } from "../shared/perf";
```
Would be printed as:
```ts
import huffman from "n-ary-huffman";
import { repr } from "tiny-decoders";

import iconsChecksum from "../icons/checksum";
import type { ElementReport, ElementTypes, ElementWithHint, ExtendedElementReport, HintMeasurements, HintUpdate } from "../shared/hints";
import { type HintsMode, type KeyboardAction, type NormalizedKeypress } from "../shared/keyboard";
import { Resets, addListener, bind, isMixedCase, log, makeRandomToken, partition, splitEnteredText, unreachable } from "../shared/main";
import type { FromBackground, FromOptions, FromPopup, FromRenderer, FromWorker, ToBackground, ToOptions, ToPopup, ToRenderer, ToWorker } from "../shared/messages";
import { type OptionsData, type PartialOptions, flattenOptions, getDefaults, makeOptionsDecoder, unflattenOptions } from "../shared/options";
import { type Durations, type Perf, TimeTracker } from "../shared/perf";
```
(example taken from the [prettier discussion](https://github.com/prettier/prettier/issues/5995#issuecomment-475125168) on this topic)

**Describe the use case you need it for**
IMHO single line imports are much cleaner and take up less space.

**Other**
Note: dprint added this feature some time ago: https://github.com/dprint/dprint-plugin-typescript/issues/365

**Comment by MichaReiser** [comment]
Rome's goal is to reduce the discussions around how code should be formatted. Supporting many options goes against that goal because it turns discussions around formatting into discussions on what formatter options to use. 

That's why Rome won't support such a requested option and recommend using `dprint` instead if you or your team is very opinionated on how to format code.

**Reply by Waterstraal** [reply]
It's a valid argument not to want to support many options. In that case, I would like to suggest to not make this an option, but to just make the Rome Formatter format imports on 1 line instead. This way, everybody will benefit from having their imports formatted in a better way, and Rome has a USP over Prettier.

**Reply by jeysal** [reply]
I'm not sure I see why the proposed formatting is a better way based on the linked issue. Even the linked comment acknowledges that it is such a controversial proposal that it should probably be behind an option if anything. I only see hardly convincing arguments like the `// @ts-ignore` one.
Rome does not try to differentiate itself from Prettier with a different formatting style, in fact formatting similarly to Prettier is a good selling point for easy adoption without massive diffs.

**Comment by ysageev** [comment]
I do agree with @Waterstraal.  I find scrolling to be an impediment to code clarity and try to minimize it.   I much prefer one line imports.  It would be nice to have small number of settings we could adjust.   For example, I also prefer

```js
const f = () => g()
```

to 

```js
const f = () =>
   g()
```
